packages/amplify-util-mock/package.json

```diff
@@ -94,7 +94,7 @@
     "graphql-versioned-transformer": "^5.2.72",
     "isomorphic-fetch": "^3.0.0",
     "jest": "^29.0.0",
-    "jsonwebtoken": "^9.0.0",
+    "jose": "^5.2.0",
```

Comment:
Unfortunately, it is not possible to use newer versions of jsonwebtoken because it is no longer supporting token signing on browsers. Please check out [this issue](https://github.com/auth0/node-jsonwebtoken/issues/863#issue-1507663709) and [this comment](https://github.com/auth0/node-jsonwebtoken/issues/863#issuecomment-1362770212) made by one of the contributors of jsonwebtoken.

I initially thought of downgrading jsonwebtoken to a version that works on browsers, but I later ended up with using a package that is meant to support browsers instead due to potential problems that might occur from version downgrading.